docs/conf/inspircd.conf.example

```diff
@@ -153,9 +153,12 @@
       type="clients"
 
       # ssl: If you want the port(s) in this bind tag to use SSL, set this
-      # to either "gnutls" or "openssl". The appropriate SSL module must be
-      # loaded for SSL to work. If you do not want the port(s) in this bind
-      # tag to support SSL, just remove or comment out this option.
+      # to "openssl", gnutls", "mbedtls" or the name of a custom <sslprofile>
+      # tag that you have defined. See the wiki page for the SSL module you are
+      # using for more details.
```

Comment:
Not quite, the "openssl" etc. tags are only intended for backwards compat so they only work if there are no sslprofiles defined (https://github.com/inspircd/inspircd/blob/master/src/modules/extra/m_ssl_openssl.cpp#L842)